forgerock-auth/src/main/java/org/forgerock/android/auth/Config.java

```diff
@@ -195,6 +198,20 @@ private CookieJar getCookieJar() {
         return cookieJar;
     }
 
+    @VisibleForTesting
+    void setSSOModel(SSOBroadcastModel ssoModel) {
+        this.ssoBroadcastModel = ssoModel;
+    }
+
+
+    SSOBroadcastModel getSSOBroadcastModel() {
+        if (ssoBroadcastModel == null) {
+            return new SSOBroadcastModel();
```

Comment:
Set the `ssoBroadcastModel` attribute with the new instance?

Reply:
Fixed it, good catch